uaclient/security_status.py

```diff
@@ -175,24 +189,37 @@ def filter_updates(
                             not counted_as_security
                             and "backports" not in expected_origin.archive
                         ):
+                            upgrade_status = (
+                                None
+                                if version == candidate_version
+                                else UpdateStatus.AVAILABLE_LOWER_PRIORITY.value
+                            )
                             result["standard-updates"].append(
-                                (version, expected_origin.site)
+                                (version, expected_origin.site, upgrade_status)
                             )
 
                 # This loop should be only used if the user does not have esm
                 # (infra or apps) enabled, and it is shorter than the
                 # previous one
                 if package.name in esm_cache:
                     esm_package = esm_cache[package.name]
+                    esm_candidate_version = policy.get_candidate_ver(
+                        esm_package
+                    )
                     for version in esm_package.version_list:
                         if version > package.current_ver:  # type: ignore
                             for origin, _ in version.file_list:
                                 service = get_origin_information_to_service_map().get(  # noqa: E501
                                     (origin.origin, origin.archive)
                                 )
                                 if service:
+                                    upgrade_status = (
+                                        None
+                                        if version == esm_candidate_version
+                                        else UpdateStatus.AVAILABLE_LOWER_PRIORITY.value
+                                    )
                                     result[service].append(
-                                        (version, origin.site)
+                                        (version, origin.site, upgrade_status)
```

Comment:
When the user does not have a service enabled, the upgrade will never be available
This part is not needed afaics